scout/server/blueprints/variants/forms.py

```diff
@@ -94,6 +94,8 @@ class VariantFiltersForm(FlaskForm):
     genetic_models = SelectMultipleField(choices=GENETIC_MODELS)
 
     cadd_score = BetterDecimalField("CADD", places=2, validators=[validators.Optional()])
+    compound_rank_score = IntegerField("Compound rank score")
+    compound_follow_filter = BooleanField("Follow filter")
```

Comment:
If I understand it correctly, the same filter settings chosen by the users are also applied to the compound vars?

At least  the settings chosen for coordinates (lt, gt), frequency (gnomad, local obs, clingen ngi and swegen), in (region_annotations, functional_annotations, clinsig, svtype, genetic_models) and spidex


<img width="595" alt="image" src="https://user-images.githubusercontent.com/28093618/158975800-334e6d2c-380a-41c8-81ea-8c254ee5533c.png">

I think this becomes a very powerful tool and the central concept should be super clear.

This form item with the tooltip don't look very self explanatory to me:

<img width="233" alt="image" src="https://user-images.githubusercontent.com/28093618/158975628-2ed4cb8b-8049-457c-b710-26b4fb14cf6e.png">

For this reason I would rename "follow filter" to something like "apply same filters to variant's compounds"

The tooltip is better, but perhaps "Shade compounds as if dismissed if they are excluded by selected filters"

Reply:
Right, but that is a veeeery long name on a button. I think the users will learn to identify what it does, but if you have another succinct suggestion that captures the functionality (I considered the likes of "emulate" or "replicate") I'm all ears! 

 I much agree on the tool tip - updated!

Reply:
> Right, but that is a veeeery long name on a button. I think the users will learn to identify what it does, but if you have another succinct suggestion that captures the functionality (I considered the likes of "emulate" or "replicate") I'm all ears!
> 
> I much agree on the tool tip - updated!

Apply variant's filter?

Reply:
I considered ˜Apply filter˜ but it has two downsides from my perspective: 1) apply is often used as a name for update-form-with-what-I-have; I would have interpreted that as use-the-rank-score-next-to-it 2) not so important, but "apply" to me has very little wiggle room for not actually applying all of the filter (such as genes and chromosomes that really makes little sense).

Reply:
`Extend variant's filter`? To me "follow filter" is not really helpful as a name..

Reply:
Nah, it doesn't extend it, rather does roughly the same only not quite the same way. So it follows, emulates, perhaps replicates or clones. It is also has context, right between the "compound rank score" and "hide compounds" buttons, so doesn't have to contain all of that. More suggestions please! 😄 

Btw I saw that it has issues with multiple in-items again, likely after the previous test fix... I'll fix and add another test.

Reply:
Fine, name it "follow filter"! :roll_eyes:

Reply:
😄 How about "Parallel filter"? One more word should fit as well. Say "According to filter"? How about not relying on context, like "Compounds follow filter" or "Shade with filter"?